keyboards/southpole/southpole.h

```diff
@@ -0,0 +1,20 @@
+#ifndef southpole_H
+#define southpole_H
+
+#include "quantum.h"
+
+#define KEYMAP( \
```

Comment:
Please change this to:

```suggestion
#define LAYOUT( \
```